docs/sdk/src/guides/your_first_pallet.rs

```diff
@@ -0,0 +1,325 @@
+//! # Currency Pallet
+//!
+//! By the end of this guide, you will write a small FRAME pallet (see
+//! [`crate::polkadot_sdk::frame_runtime`]) that is capable of handling a simple crypto-currency.
```

Comment:
Super minor thing, not important.. But I think it's more consistent with the rest of the crypto ecosystem (Eth, Solana) to use the terminology token rather than currency for this type of functionality.

If possible we should not use the terms interchangeably; currency generally means something used as a means of exchange, where token is more general and could be many things (like used for governance, securities ownership, or means of exchange).

```suggestion
//! # Token Pallet
//!
//! By the end of this guide, you will write a small FRAME pallet (see
//! [`crate::polkadot_sdk::frame_runtime`]) that is capable of handling a simple token.
```